### PR TITLE
Configure HTTP_PORT and WS_PORT via environment variables

### PR DESCRIPTION
**Pull request by Charles-Johnson**
This should resolve issue #2829.

### CODE REVIEW

File: node/src/opt.rs [hunk 1]
```diff
@@ -124,6 +125,7 @@ pub struct Opt {
         long,
         default_value = "8020",
         value_name = "PORT",
+        env = "WS_PORT",
```
Comment (maoueh) [inline node/src/opt.rs:128]: This seems wrongly place. This is for the JSON-RPC of the admin API ...

**Comment by Charles-Johnson** [comment]
Sorry @maoueh , I've been occupied by work and family for the last 2 weeks. I need still need to find some time to understand the proper fix

**Comment by maoueh** [comment]
@Charles-Johnson Didn't wanted to put pressure here, sorry if it felt that way. Just wanted to ensure this was still active :)

**Comment by maoueh** [comment]
Sorry for such long delay, completely forgot. Would you have time to change `HTTP_PORT` to `GRAPH_GRAPHQL_HTTP_PORT` and `WS_PORT` `GRAPH_GRAPHQL_WS_PORT` just to bring a bit more clarity about the usage of those two and add quick list items in https://github.com/graphprotocol/graph-node/blob/master/docs/environment-variables.md#graphql about those new 2.

**Comment by otaviopace** [comment]
@Charles-Johnson @maoueh will this be continued? Otherwise I would "vote" for closing it.

**Comment by maoueh** [comment]
@otaviopace I'll take care of bringing it in.

**Comment by Charles-Johnson** [comment]
@maoueh just looking into the quick list items now

**Comment by Charles-Johnson** [comment]
@maoueh I made the requested changes

**Review by leoyvens — APPROVED** [review]
Looks like the review was addressed, thanks @Charles-Johnson

**Pull request merged**